### ISSUE TITLE
Amoco

### ISSUE THREAD

**Issue by maxerickson** [osmlab/name-suggestion-index#6292]
Wikidata currently conflates the defunct company with the brand, so the NSI entries are suppressed by the check for things that are marked dissolved in Wikidata.

**Comment by EOEboh** [comment]
what country is it based?

**Comment by 1ec5** [comment]
Yes, the item needs to be split into two items, one for the former company and the other for the current brand that was formerly owned by it.
